javascript/javascript2/week1/shortestWord.js

```diff
@@ -0,0 +1,16 @@
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium", ""];
+
+const findshortestWord = function(array){
+    let shortestWordInArray = array[0];
+
+    for(let word of array){
+        if(word.length !== 0){
+            if(word.length < shortestWordInArray.length){
```

Comment:
When nesting if-statements like this without an else-case, you can just use one if-statement:
```js
if(word.length && word.length < shortestWordInArray.length) {
```
I also removed the `!== 0` part because 0 is a falsy value in JavaScript.

Reply:
yes  the way you did , its really nice, i will do that . but  can we use this one also if(word.length) > 0 && the second test condition